src/variety/slither.js

```diff
@@ -266,6 +304,23 @@
 			this.checkAllCell(function(cell) {
 				return cell.qnum >= 0 && cell.getdir4BorderLine1() !== cell.qnum;
 			}, "nmLineNe");
+		},
+
+		checkdir4VertexLine: function() {
```

Comment:
I think this check was only used by Vertex Slitherlink, so it's not needed in this file.

Reply:
Removed